src/main/java/com/shair13/external_service/security/SwaggerConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.shair13.external_service.security;
+
+import com.shair13.external_service.configuration.UrlConfig;
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import io.swagger.v3.oas.models.servers.Server;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+@RequiredArgsConstructor
+public class SwaggerConfig {
+
+    private final UrlConfig urlConfig;
+    @Bean
+    public OpenAPI customOpenAPI() {
+        return new OpenAPI()
```

Comment:
Dla uproszczenia wywoływania API, warto jest skonfigurować możliwość uwierzytelnienia za pomocą loginu i hasła, a nie tylko tokenu JWT.

Reply:
Dodałem możliwość uwierzytelnienia za pomocą username i password. 
testowy użytkownik to user - password.